site/src/github.rs

```diff
@@ -654,7 +654,7 @@ async fn summarize_run(
 
     for (title, stat, hidden, comparison) in stats {
         message.push_str(&format!(
-            "## [{title}]({})\n",
+            "\n## [{title}]({})\n",
```

Comment:
I think the more correct fix is to ensure that each stat has a blank line after it no? 

```suggestion
            "## [{title}]({})\n\n",
```

Reply:
Did this fix the broken comment in the PR? From my experience, it's fine if a header doesn't have a blank line after it in Markdown, so I thought that the problem is that there's no blank line between the details and the next header. But if this fixed the comment then sure, let's do this. I couldn't fix the comment manually because I don't have rights for it, so I couldn't experiment with it.

Reply:
Ah no you're right. I think it's that the `</details>\n` closing tag needs to be `</details>\n\n`.

Reply:
I also think we should remove the `**Summary:**` line.

It would be good if we could have tests for all of this...

Reply:
Tests would be nice, I'll try to think about how we can separate the rendering from providing data to enable tests.